packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -336,11 +295,12 @@ exports.createContentTypeNodes = ({
         }
       })
 
-      // Add reverse linkages if there are any for this node
+      // // Add reverse linkages if there are any for this node
       const foreignReferences = foreignReferenceMap[entryItem.sys.id]
       if (foreignReferences) {
         foreignReferences.forEach(foreignReference => {
           const existingReference = entryItemFields[foreignReference.name]
+          // console.log(`adding backreference`, foreignReference)
```

Comment:
I think we can remove this too

Reply:
Thanks, yeah, need to remove that stray console.log (relict of debugging)

Reply:
Removed